src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedInstanceMethodSymbol.cs

```diff
@@ -30,11 +30,11 @@ public sealed override bool IsImplicitlyDeclared
             }
         }
 
-        public override bool AreLocalsZeroed
+        public sealed override bool AreLocalsZeroed
```

Comment:
>public sealed override bool AreLocalsZeroed [](start = 8, length = 43)

Are we certain that we want an attribute on a type to affect all methods that compiler synthesizes? Strictly speaking it is not user code, why should it be affected? Depending on the answer, we probably better of implementing this method in each leaf class, so that we can differentiate the behavior.

Reply:
We tentatively decided yes when designing this feature, but I think that's a reasonable question.